### PR TITLE
Update homework.sh

### PR DESCRIPTION
**Pull request by FredyLRincon**
Hi Simeon! @DTXE

This is my HOMEWORK submission for feedback.
This is my first GitHub submission EVER! 

I hope it meets with the expectations and requirements.

Looking forward for your comments.

Fredy

## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding code
## What did you learn from the changes you have made?
Learn how to use echo, cd, >, mkdir, rm, ls and to make a file executable (chmod)
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
There were many ways of completing the homework. The easiest was to work from the PARENT folder
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
No challenges, but I learned that gitbash can support multiple commands in one line
## How were these changes tested?
Using gitbash
## A reference to a related issue in your repository (if applicable)
N/A
## @mentions of the person or team responsible for reviewing proposed changes (At least 2 people)
- @DTXE

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

**Comment by FredyLRincon** [comment]
@DTXE, I am hoping this first submission meets with the requirements of the program and that I am not breaking anything.

Cheers,

Fredy

File: 03_homework/homework.sh [hunk 1]
```diff
@@ -7,21 +7,33 @@
...
+cd FLRincon05 ; rm Love4Data05.txt ; cd ..
 
 # 8. How would you navigate back to the parent directory containing all the directories?
-
+cd ..
 # 9. How would you remove each directory along with its contents?
-
+rm -r FLRincon01 FLRincon02 FLRincon03 FLRincon04 FLRincon05
 # 10. How would you verify that all directories and files have been deleted?
+ls -R
```
Comment (dwz92) [inline 03_homework/homework.sh]: `ls -al` is fine, no need to search recursively.

**Review by dwz92 — APPROVED** [review]
good

**Review by FredyLRincon — COMMENTED** [review]
This is revision # 2
Question # 10 has been corrected as requested.

Thanks for your feedback!

**Pull request merged**